api/db/seeds/data/organizations-builder.js

```diff
@@ -11,13 +11,13 @@ module.exports = function organizationsBuilder({ databaseBuilder }) {
     organizationId: 2,
     organizationRole: Membership.roles.ADMIN,
   });
-
   databaseBuilder.factory.buildOrganization({
     id: 3,
     type: 'SCO',
     name: 'The Night Watch',
     isManagingStudents: true,
     canCollectProfiles: true,
+    email: 'generic.account@sco.net'
```

Comment:
En général, on utilise des adresses en `@example.net`

Reply:
c'est juste, c'était pour signaler que c'était une SCO, que penses-tu de `generic.account@sco.example.net` ?

Reply:
The Night Watch est de type 'SCO'

Reply:
done